add a `--no-install` option to the extensions-sdk cli to skip dep install (#20752)
directus/directus: .changeset/young-items-brush.md

---

> '@directus/extensions-sdk': patch

Reviewer: ```suggestion
'@directus/extensions-sdk': minor
'create-directus-extension': minor
```

---

> Added a --no-install option to the extensions-sdk cli to skip dep install

Reviewer: ```suggestion
Added a `--no-install` option to the extensions CLI to skip dependency installation
```

---
'@directus/extensions-sdk': minor
'create-directus-extension': minor
---

> Added a `--no-install` option to the extensions CLI to skip dependency installation

Reviewer: ```suggestion
Added a `--no-install` option to the extensions CLI allowing to skip dependency installation
```
